enacts/wat_bal/maproom_monit.py

```diff
@@ -470,10 +460,16 @@ def wat_bal_tile(tz, tx, ty):
     )
 
     mymap_min = 0
-    mymap_max = 8* (int(taw_max.values) // 8)
+    mymap_max = CONFIG["taw_max"]
     mycolormap = CMAPS["precip"]
 
-    taw_tile = taw.sel(
+    garbage, taw_tile = xr.align(
```

Comment:
It's conventional to use the name `_` (underscore) for a return value you don't intend to use.